src/resolve-host/resolve-host.c

```diff
@@ -64,10 +64,12 @@ static void print_source(uint64_t flags, usec_t rtt) {
         fputs("\n-- Information acquired via", stdout);
 
         if (flags != 0)
-                printf(" protocol%s%s%s",
+                printf(" protocol%s%s%s%s%s",
```

Comment:
That's fair, but I'd probably be using fprintf(stdout, ) instead of printf, for the sake of consistency with the surrounding prints.